### PR TITLE
[RELEASE] 20170214

### PR DESCRIPTION
**Pull request by MiguelMoll**
- closes #1667 Add "Accept: application/json" header to client requests [ @soulshake ]
- closes #1677 Dedicated build instance [ @ddollar ]
- closes #1907 Fix forks failing to build on Travis [ @spencerhakim ]
- closes #1919 Use convox/init for new apps [ @MiguelMoll ]
- closes #1922 Update api/manifest.yml [ @soulshake ]
- closes #1923 Add stdcli Writer.Warn() [ @soulshake ]
- closes #1924 guard against nil values from old task definitions [ @ddollar ]
- closes #1925 clarify usage statements for --id [ @soulshake ]
- closes #1929 dont send events in test environment [ @nzoschke ]

## Milestone Release
- [x] Release branch
- [x] Pass CI
- [x] Code review
- [x] Merge into master
- [ ] Close milestone
- [ ] Release master
- [ ] Record release number: 
- [ ] Pass CI
- [ ] Update demo
- [ ] Deploy demo/httpd
- [ ] Deploy demo/rails
- [ ] Update staging
- [ ] Deploy staging/site-staging
- [ ] Deploy staging/console-staging
- [ ] Update [release notes](https://github.com/convox/rack/releases)
- [ ] Documentation review
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[20170214](https://trello.com/c/QgO6B2VI/1917-20170214)

File: cmd/convox/init_test.go [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+}
+
+func TestGenerateManifest(t *testing.T) {
+	manifestTests := []struct {
+		pf Procfile
+		af Appfile
+		r  Release
+		m  manifest.Manifest
+	}{
+		{ //
```
Comment (nzoschke) [inline cmd/convox/init_test.go]: unnecessary comments?

File: provider/aws/dist/rack.json [hunk 2]
```diff
@@ -1249,7 +1270,6 @@
       "Properties": {
         "GroupDescription": "Instances",
         "SecurityGroupIngress": [
-          { "IpProtocol": "tcp", "FromPort": "22", "ToPort": "22", "CidrIp": { "Ref": "VPCCIDR" } },
```
Comment (nzoschke) [inline provider/aws/dist/rack.json]: which patch did this come in with? might this affect anyone that set up real ssh access?

File: cmd/convox/init_test.go (hunk 1 above)
Comment (MiguelMoll) [inline cmd/convox/init_test.go]: I was using `//` as a beginning/end block for input and output for a given test.

File: provider/aws/dist/rack.json (hunk 2 above)
Comment (MiguelMoll) [inline provider/aws/dist/rack.json]: This came with the dedicated build changes: https://github.com/convox/rack/pull/1677/files#diff-54ede314b4e7f9fbc11373ad5f9b8bc2L1252

Comment (ddollar) [inline provider/aws/dist/rack.json]: This should be covered by the line below it.

Comment (nzoschke) [inline provider/aws/dist/rack.json]: got it

**Pull request merged**